perfkitbenchmarker/flag_util.py

```diff
@@ -190,3 +191,149 @@ def DEFINE_integerlist(name, default, help, on_nonincreasing=None,
   serializer = IntegerListSerializer()
 
   flags.DEFINE(parser, name, default, help, flag_values, serializer, **args)
+
+
+SI_SUFFIXES = ['', 'K', 'M', 'G', 'T', 'P']
+
+
+@functools.total_ordering
+class ObjectSize(object):
+  """Holds the size of data, i.e. numbers of bytes.
+  """
+
+  def __init__(self, bytes=0):
+    self.bytes = bytes
+
+  def mult_for_base_and_exp(self, base, exp):
+    """Given base and exp, returns mult so that
+    bytes = mult * base ^ exp.
+
+    May return a float.
+    """
```

Comment:
Since this is a multi-line docstring, write it in standard style with the "Args:" and "Returns:" sections.